src/utils/ddb/ddb_vos.h

```diff
@@ -48,8 +48,18 @@ struct ddb_array {
 };
 
 /* Open and close a pool for a ddb_ctx */
-int dv_pool_open(char *path, daos_handle_t *poh);
+int
+    dv_pool_open(const char *path, daos_handle_t *poh, uint32_t flags);
```

Comment:
Please adjust related format.

Reply:
It is a bit annoying, this is auto-format by clang format, let me fix this.

Reply:
git hooks required this..i leave it for now.